React/tic-tac-toe/src/app/screens/Login/fields/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import cx from 'classnames';
+
+const getValidityClassName = meta => {
```

Comment:
you should not need to use this. Directly use and define styles in a sass file so you can call it and change the aspect. Also, the sass file should be local not general for all the app

Reply:
sass modules?